climesync.py

```diff
@@ -34,12 +37,86 @@
     "h - print this menu\n"
     "q - exit\n")
 
-arg_username = ""
-arg_password = ""
-timesync_url = ""
 ts = None  # pymesync.TimeSync object
 
 
+def validate_config(path="~/.climesyncrc"):
+    """Ensure that the configuration file is formatted correctly"""
+
+    realpath = os.path.expanduser(path)
+
+    # Check that the file exists. Using a non-file object is non permitted.
+    if not os.path.isfile(realpath):
+        return False
+
+    # A pound sign followed by any number of characters
+    comment = re.compile("#.*")
+
+    # Key name, equals sign, value
+    # Keys and values can consist of any character except an equals sign
+    config = re.compile("[^=]+ = [^=]+")
+
+    with open(realpath, "r") as f:
```

Comment:
Why are we writing regexes to parse our config file instead of just using a library like https://docs.python.org/3.5/library/configparser.html ?